pkg/reconciler/pipelinerun/resources/pipelineref_test.go

```diff
@@ -1154,42 +1164,27 @@ func TestGetPipelineFunc_GetFuncError(t *testing.T) {
 	requesterUnsigned := test.NewRequester(resolvedUnsigned, nil)
 	resolvedUnsigned.DataErr = fmt.Errorf("resolution error")
 
-	prBundleError := &v1beta1.PipelineRun{
+	prResolutionError := &v1.PipelineRun{
 		ObjectMeta: metav1.ObjectMeta{Namespace: "trusted-resources"},
-		Spec: v1beta1.PipelineRunSpec{
-			PipelineRef: &v1beta1.PipelineRef{
-				Name:   "pipelineName",
-				Bundle: "bundle",
-			},
-			ServiceAccountName: "default",
-		},
-	}
-
-	prResolutionError := &v1beta1.PipelineRun{
-		ObjectMeta: metav1.ObjectMeta{Namespace: "trusted-resources"},
-		Spec: v1beta1.PipelineRunSpec{
-			PipelineRef: &v1beta1.PipelineRef{
+		Spec: v1.PipelineRunSpec{
+			PipelineRef: &v1.PipelineRef{
 				Name: "pipelineName",
-				ResolverRef: v1beta1.ResolverRef{
+				ResolverRef: v1.ResolverRef{
 					Resolver: "git",
 				},
 			},
-			ServiceAccountName: "default",
+			TaskRunTemplate: v1.PipelineTaskRunTemplate{
+				ServiceAccountName: "default",
+			},
 		},
 	}
 
 	testcases := []struct {
 		name        string
 		requester   *test.Requester
-		pipelinerun v1beta1.PipelineRun
+		pipelinerun v1.PipelineRun
 		expectedErr error
 	}{
-		{
-			name:        "get error when oci bundle return error",
```

Comment:
would it make sense to replace this test with a bundle resolver? @Yongxuanzhang I'm sort of confused what this is supposed to test based on the error message "failed to get keychain", but it seems like something we'd want to maintain coverage for-- are you able to give a bit more detail? (Same comment for the related taskref test)

Reply:
Yes this is for the test coverage for the error from bundle here: https://github.com/tektoncd/pipeline/blob/06d249921f48c5351a7eb9560fd87fac067c2ef9/pkg/reconciler/pipelinerun/resources/pipelineref.go#L76 
I think it's ok to remove this test since the code is removed in this PR